src/util/helm.js

```diff
@@ -49,11 +51,23 @@ const updateHelmChartVersion = async (chartPath, version) => {
     await fs.writeFile(chartYamlFile, doc.yaml);
 };
 
+const extractChartUrl = async (chartPath) => {
+    //extract dependency chart URL
+    const chartYamlFile = path.join(chartPath, "Chart.yaml");
+    const chartYaml = await fs.readFile(chartYamlFile);
+
+    const doc = new YAML(chartYaml.toString());
+
+    return doc.dependencies.repository;
```

Comment:
Isn't `dependencies` an array?

Reply:
yea, we realized this later on. but now we're extracting information from the chart in a different way